expandable_textview/src/main/res/values/attrs.xml

```diff
@@ -5,6 +5,7 @@
         <attr name="originalText" format="string" />
         <attr name="expandAction" format="string" />
         <attr name="expandActionColor" format="string" />
+        <attr name="lessMoreAction" format="string" />
```

Comment:
the name is not really descriptive, a better one is `collapseAction` to align with `expandAction`.